### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by NatanyaDC**
Added my CV

### CODE REVIEW

File: student-work/2023/Natanya-DaCosta/readme.md [hunk 1]
```diff
@@ -1 +1,96 @@
...
++972 (0)55-290-7070  
+Dolev, Israel  
+enatanyadc@gmail.com  
+[See my portfolio](https://enatanyadc.wixsite.com/natanya)
+
+## Experience
+### _Communications Manager_ | B’nai Brith New Zealand
+2018–2022  
+
+Created content for a non-profit.
```
Comment (lsnovich) [inline student-work/2023/Natanya-DaCosta/readme.md:11]:
```suggestion
* Created content for a non-profit.  
```

Comment (lsnovich) [reply student-work/2023/Natanya-DaCosta/readme.md:11]: Please add bullets like this to the rest of the content that needs it

File: student-work/2023/Natanya-DaCosta/readme.md [hunk 2]
```diff
@@ -1 +1,96 @@
+# Natanya Da Costa
++972 (0)55-290-7070  
+Dolev, Israel  
+enatanyadc@gmail.com
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Natanya-DaCosta/readme.md:4]:
```suggestion
✉️ enatanyadc@gmail.com  
```

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
please make the change

**Review by lsnovich — APPROVED** [review]
Looks great!
Merging now. Please take a screenshot of the merge message and upload it to Google Classroom.

**Review by Laura-Novich-OBW — APPROVED** [review]

**Pull request merged**